packages/genesys-spark-components/src/components/beta/gux-side-panel/components/gux-side-panel-heading/gux-side-panel-heading.scss

```diff
@@ -0,0 +1,23 @@
+:host {
+  display: block;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  display: flex;
+  flex: 0 0 auto;
+  gap: var(--gse-ui-sidePanel-header-iconGap);
+  align-items: center;
+  width: 100%;
```

Comment:
might just be worth updating with main when you get a chance and running `npm run lint` again to update these to logical properties.